src/action.ts

```diff
@@ -182,7 +185,8 @@ function unlinkMetafile(meta: Metafile, destinationDir: string) {
 	}
 	const fullPath = join(destinationDir, rootFolder);
 	if (!existsSync(fullPath)) return;
-	if (resolve(fullPath) === resolve(destinationDir)) return;
+	if (!fullPath.startsWith(destinationDir)) return; // assert: fullPath is within destinationDir
+	if (statSync(fullPath).ino === statSync(destinationDir).ino) return; // assert: fullPath is not destinationDir
```

Comment:
I have never seen this happen or ever expect it to. There is no way to unlink everything outside this method.

Reply:
In that case I think [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) applies

Reply:
I'm just paranoid as this is the only place where cross-seed can be dangerous. These checks guarantees that we are deleting the right content path, instead of hundreds or thousands of user torrent data.

I think it's better to have it and not need it, rather than need it but not have it.

Reply:
Good reasoning. Let's leave it in